lib/xcodeproj/project/object_attributes.rb

```diff
@@ -354,14 +354,15 @@ def has_one(singular_name, isas)
             # 1.9.2 fix, see https://github.com/CocoaPods/Xcodeproj/issues/40.
             public(attrb.name)
 
+            variable_name = "@#{attrb.name}"
```

Comment:
The attribute method definitions were allocating over 360k instances of "@dependencies" and "dependencies", so I thought this was worth it :)

Reply:
nit: `variable_name = :"@#{attrb.name}"` so it's a symbol instead of a string

Reply:
fixed